src/website/src/app/documentation/demos/forms/ng/generic.html

```diff
@@ -0,0 +1,60 @@
+<!-- Template Driven Example -->
+<form clrForm>
+  <input clrControl placeholder="Basic text" name="basic" [(ngModel)]="model.basic" />
```

Comment:
I thought we wanted to only show examples with a label (same comment for lines 6, 34, & 37) b/c thats the guidance we give for forms.

Reply:
Good point, I'll remove the line.